python/taichi/lang/__init__.py

```diff
@@ -77,6 +77,14 @@ def init(default_fp=None, default_ip=None, print_preprocessed=None, debug=None,
 
   for k, v in kwargs.items():
     setattr(ti.cfg, k, v)
+
+  if os.environ.get("TI_PRINT_IR") is not None:
```

Comment:
What if the user does `export TI_PRINT_IR=0`?

Reply:
if user want `print_ir = False`, they're expected to `unset TI_PRINT_IR`.